tokio/src/runtime/task/core.rs

```diff
@@ -36,13 +37,37 @@ pub(super) struct Cell<T: Future, S> {
     pub(super) trailer: Trailer,
 }
 
+/// Uninitialized task cell. Only the holds a valid future.
+///
+/// `UninitCell<T, S>` should have same layout as `Cell<T, S>`, so it can
+/// be initialized and transmute into a `Cell<T, S>` lately.
```

Comment:
```suggestion
/// `UninitCell<T, S>` should have the same layout as `Cell<T, S>`, so it can
/// be initialized and transmute into a `Cell<T, S>` later.
```